content/en/docs/concepts/cluster-administration/node-shutdown.md

```diff
@@ -0,0 +1,274 @@
+---
+title: Node Shutdowns
+api_metadata:
+- kind: "Node"
```

Comment:
The `api-metadata` section needs a `apiVersion` field to be complete. Let's either add it in _(preferred option)_ or consider removing the `api-metadata` section altogether from this new page.

 Below is a suggestion on how it would look with the addition.
 
```suggestion
api_metadata:
- apiVersion: "v1"
  kind: "Node"
```

Reply:
Done. Thanks